Osiris/SDK/WeaponId.h

```diff
@@ -173,5 +184,12 @@ constexpr int getWeaponClass(WeaponId weaponId) noexcept
     case WeaponId::Awp:
     case WeaponId::G3SG1:
     case WeaponId::Scar20: return 38;
+
+    case WeaponId::Flashbang:
+    case WeaponId::Decoy:
+    case WeaponId::HeGrenade:
+    case WeaponId::IncGrenade:
+    case WeaponId::Molotov:
+    case WeaponId::SmokeGrenade: return 40;
```

Comment:
You dont need to do this gigant meme i did a few months back, just checking if the activeweapon is a grenade works
```
    constexpr auto isGrenade() noexcept
    {
        return getWeaponType() == WeaponType::Grenade;
    }
```

Reply:
Thanks, that's a good point

Reply:
So
`  constexpr auto isGrenade() noexcept
    {
        return getWeaponType() == WeaponType::Grenade;
    }`

Is all I need? Adding all the code to test the ragebot